test/libsolidity/semanticTests/smoke_test.sol

```diff
@@ -84,11 +87,14 @@ contract C {
         return (["any", "any"], ["any", "any", "any"]);
     }
 }
+// ====
+// EVMVersion: >homestead
 // ----
 // constructor(): 3 ->
 // state() -> 3
 // _() -> FAILURE
-// e() ->
+// d() ->
+// e() -> FAILURE, hex"08c379a0", 0x20, 19, "Transaction failed."
```

Comment:
Can you also add an example with a string longer than 32 bytes?

Reply:
Supporting strings longer than 32 bytes is a bit more work and out-of-scope, I'd say. Created an issue for it: https://github.com/ethereum/solidity/issues/7197.